### PR TITLE
[qt] Fix warning message

### PR DESCRIPTION
**Pull request by MartinDelille**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

There is currently a warning when compiling Qt models:
```
/path/to/client/SWGApiResponse.h:39:10: warning: 'asJson' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
        QString asJson ();
                ^
```

These PR fix it.

Ping @ravinikam @stkrwork @fvarose

### CODE REVIEW

**Comment by etherealjoy** [comment]
Looks good to me, good increments to improve the qtcpp generator.
@MartinDelille Are you working with a complex yaml/json api file?

**Comment by MartinDelille** [comment]
@etherealjoy Not yet but I'm planning too. Unfortunately I have a blocking issue there: #7802

File: modules/swagger-codegen/src/main/resources/qt5cpp/model.mustache [hunk 1]
```diff
@@ -38,13 +38,13 @@ public:
 
     void cleanup();
 
-    String asJson ();
+    String asJson () override;
```
Comment (stkrwork) [inline modules/swagger-codegen/src/main/resources/qt5cpp/model.mustache]: I dont think this file is actually needed, it doesnt use anything from QT and has a tizen namespace defined. Can you try generating a qt client without this mustache file? (Also remove it from the java file, if necessary)

Comment (MartinDelille) [reply modules/swagger-codegen/src/main/resources/qt5cpp/model.mustache]: Yes indeed. I just removed it.

**Comment by wing328** [comment]
@MartinDelille thanks for the PR, which has been merged into master.

Thanks @etherealjoy @stkrwork for reviewing the change.

**Pull request merged**